tests/performance/density-kwok.go

```diff
@@ -0,0 +1,226 @@
+package performance
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	kubevirtv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/libvmi"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libvmifact"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+const nodeCount = 100
+
+var _ = FSIGDescribe("[KWOK] Control Plane Performance Density Testing using kwok", func() {
+	var (
+		kubevirtClient kubecli.KubevirtClient
+		k8sClient      *kubernetes.Clientset
+		startTime      time.Time
+	)
+
+	artifactsDir, _ := os.LookupEnv("ARTIFACTS")
+
+	BeforeEach(func() {
+		skipIfNoKWOKPerformanceTests()
```

Comment:
We should rather fail and use labels in order to filter/focus

Reply:
Since, the other density tests are following the same approach based on flags, used the same here for consistency.

Reply:
+1 for @xpivarc's suggestion